registry/registry.go

```diff
@@ -130,9 +131,23 @@ func (registry *Registry) ListenAndServe() error {
 			},
 		}
 
-		tlsConf.Certificates[0], err = tls.LoadX509KeyPair(config.HTTP.TLS.Certificate, config.HTTP.TLS.Key)
-		if err != nil {
-			return err
+		if config.HTTP.TLS.LetsEncrypt.CacheFile != "" {
```

Comment:
I would vote for mutually exclusive. I can't think of a valid reason to configure both. It seems like it would be a mistake that we should call out.